websocketOutportDriver/sender/webSocketSender.go

```diff
@@ -72,13 +71,22 @@ func NewWebSocketSender(args WebSocketSenderArgs) (*webSocketSender, error) {
 }
 
 // AddClient will add the client to internal maps and will also start
-func (w *webSocketSender) AddClient(wss WSConn, remoteAddr string) {
+func (w *webSocketSender) AddClient(wss outportData.WSConn, remoteAddr string) {
+	if wss == nil {
```

Comment:
if you add IsInterfaceNil in WSConn you can use it here

Reply:
explained above